AElf.Cryptography.Tests/AElfKeyStoreTests.cs

```diff
@@ -96,9 +98,13 @@ public async Task Open_Account_WithTimeout()
             var address = Address.FromPublicKey(keyPair.PublicKey);
             var addString = address.GetFormatted();
 
-            //Open account
-            var errResult = await _keyStore.OpenAsync(addString, "123", false);
-            errResult.ShouldBe(AElfKeyStore.Errors.None);
+            //Open account with timeout
+            _keyStore.DefaultTimeoutToClose = TimeSpan.FromMilliseconds(200);
+            _keyStore.OpenAsync(addString, "123").Wait();
+            
+            Thread.Sleep(1000);
```

Comment:
do not make unit test wait such long time

Reply:
have changed to 60ms.